substrate/frame/proxy/src/lib.rs

```diff
@@ -664,6 +671,17 @@ pub mod pallet {
 			proxy_type: T::ProxyType,
 			disambiguation_index: u16,
 		},
+		/// A pure proxy was killed by its spawner.
+		PureKilled {
+			// The pure proxy account that was destroyed.
+			pure: T::AccountId,
+			// The account that created the pure proxy.
+			spawner: T::AccountId,
+			// The proxy type of the pure proxy that was destroyed.
+			proxy_type: T::ProxyType,
+			// The index originally passed to `create_pure` when this pure proxy was created.
+			disambiguation_index: u16,
+		},
```

Comment:
is there no habit to put the variant last to avoid breaking the index of events below?

Reply:
Not that I'm aware of.

Reply:
metadata includes codec format so this won't break JS side as long as it is referencing events by name instead of index